Source/Charts/Renderers/PieChartRenderer.swift

```diff
@@ -440,7 +440,14 @@ open class PieChartRenderer: DataRenderer
 
                     if dataSet.valueLineColor != nil
                     {
-                        context.setStrokeColor(dataSet.valueLineColor!.cgColor)
+                        if dataSet.useValueColorForLine
```

Comment:
This logic seems weird to me. Why do we only get to here if `dataSet.valueLinColor != nil`?
I would think something more like:
```
DrawLine: do {
    if dataSet.useValueColorForLine {
        context.setStrokeColor(dataSet.color(atIndex: j).cgColor)
    } else if let valueLineColor = dataSet.valueLineColor {
        context.setStrokeColor(valueLineColor.cgColor)
    } else {
        return
    }
    context.setLineWidth(dataSet.valueLineWidth)

    context.move(to: CGPoint(x: pt0.x, y: pt0.y))
    context.addLine(to: CGPoint(x: pt1.x, y: pt1.y))
    context.addLine(to: CGPoint(x: pt2.x, y: pt2.y))

    context.drawPath(using: CGPathDrawingMode.stroke)
}
```

Reply:
@AlexeiGitH would take a look?

Reply:
@liuxuan30 Fixed as suggested